docs/docs/installation.md

```diff
@@ -15,8 +17,9 @@ docker run --rm ghcr.io/input-output-hk/hydra-node --help
 
 ### Prebuilt binaries
 
-We provide statically linked (as much as possible) binaries of `hydra-node` and `hydra-tui` for x86_64 Linux and ARM64 MacOS platforms as attachment of our [github releases](https://github.com/input-output-hk/hydra/releases).
+For users who prefer a traditional installation or do not use Docker, prebuilt binaries are available. We provide prebuilt binaries of `hydra-node` and `hydra-tui` for both x86_64 Linux and ARM64 MacOS platforms. These binaries are statically linked (as much as possible) to ensure ease of use across different systems. For more information, see our [github releases](https://github.com/input-output-hk/hydra/releases).
```

Comment:
This is way longer but does not add any information.

Reply:
Agree, suggest a slightly revised shorter version:

We provide statically linked binaries of `hydra-node` and `hydra-tui` for x86_64 Linux and ARM64 MacOS platforms. These binaries are available as attachments in our [GitHub releases](https://github.com/input-output-hk/hydra/releases).

Reply:
Updated